src/test/java/org/prebid/server/bidder/beachfront/BeachfrontBidderTest.java

```diff
@@ -239,6 +243,52 @@ public void makeHttpRequestsShouldAdditionalCookieHeaderForVideoRequestWhenBuyer
                         tuple(HttpUtil.COOKIE_HEADER.toString(), "__io_cid=4125"));
     }
 
+    @Test
+    public void makeHttpRequestsShouldReturnExpectedBannerRequestWithSchain() {
+        // given
+        final ExtRequestPrebidSchainSchainNode globalNode = ExtRequestPrebidSchainSchainNode.of(
+                "pbshostcompany.com", "00001", null, null, null, null, null);
+        final ExtRequestPrebidSchainSchain expectedSchain = ExtRequestPrebidSchainSchain.of(
+                null, null, singletonList(globalNode), null);
+
+        final BidRequest bidRequest = givenBidRequest(
+                impBuilder -> impBuilder
+                        .video(null)
+                        .banner(Banner.builder()
+                                .format(singletonList(Format.builder().w(100).h(300).build()))
+                                .build())
+                        .secure(1),
+                requestBuilder -> requestBuilder
+                        .source(Source.builder()
+                                .ext(ExtSource.of(
+                                        expectedSchain
+                                )).build())
+                        .user(User.builder().id("userId").buyeruid("buid").build())
+                        .device(Device.builder().model("3310").os("nokia").build()));
+
+        // when
+        final Result<List<HttpRequest<Void>>> result = beachfrontBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), BeachfrontBannerRequest.class))
+                .containsOnly(BeachfrontBannerRequest.builder()
+                        .slots(singletonList(BeachfrontSlot.of("123", "appId", BigDecimal.ONE,
+                                singletonList(BeachfrontSize.of(100, 300)))))
+                        .secure(1)
+                        .deviceModel("3310")
+                        .deviceOs("nokia")
+                        .isMobile(1)
+                        .user(User.builder().id("userId").buyeruid("buid").build())
+                        .adapterVersion("0.9.2")
+                        .adapterName("BF_PREBID_S2S")
+                        .requestId("153")
+                        .real204(true)
+                        .schain(expectedSchain)
+                        .build());
```

Comment:
Pls use containsExactly, and also we don't need to check whole BeachfrontBannerRequest model, just schain enough